### PR TITLE
Tooling Fixes

### PR DESCRIPTION
**Pull request by tobifalk**

### CODE REVIEW

File: plugins/vtd/bin/vtd-launch [hunk 1]
```diff
@@ -247,6 +255,9 @@ def start(
     pane = window.attached_pane
     pane.clear()
 
+    if opt.log_file:
+        log_file = shlex.quote(str(opt.log_file))
+        pane.send_keys(f"script {log_file}")
```
Comment (cassava) [inline plugins/vtd/bin/vtd-launch:260]: The `tmux` and `script` tools are very similar, there must be a better way than wrapping one in the other.

File: Makefile.package [hunk 2]
```diff
@@ -307,7 +308,8 @@ ${BUILD_CONANINFO}: ${SOURCE_CONANFILE}
 	# Install package dependencies and prepare in-source build.
 	#
 	mkdir -p ${BUILD_DIR}
-	conan install . --install-folder=${BUILD_DIR} \
+	conan install . ${PACKAGE_FQN} \
```
Comment (cassava) [inline Makefile.package:311]: We should add `${PACKAGE_FQN}` to all the other places we forgot, such as `${SOURCE_DIR}`, `all`, `${BUILD_CONANINFO}`, and `${BUILD_CMAKECACHE}`, right?

File: plugins/vtd/conanfile.py [hunk 3]
```diff
@@ -65,5 +65,13 @@ def package(self):
         cmake.install()
         self.copy("vtd-launch", dst="bin", src=f"{self.source_folder}/bin")
 
+    def package_id(self):
+        # Changes in a dependency's package_id need to be accounted for since
+        # the package statically links against vtd-object-lib.
+        self.info.requires["cloe-models"].full_package_mode()
```
Comment (cassava) [inline plugins/vtd/conanfile.py:71]: If we do this here, do we need to do this in other plugins too?

**Review by cassava — CHANGES_REQUESTED** [review]

File: plugins/vtd/conanfile.py (hunk 3 above)
Comment (tobifalk) [reply plugins/vtd/conanfile.py:71]: I think the special situation of the vtd plugin is the static library that is built to have the object files available also for testing. This is not done in other plugins as far as I know.

File: plugins/vtd/bin/vtd-launch (hunk 1 above)
Comment (tobifalk) [reply plugins/vtd/bin/vtd-launch]: That's right, it just took me a while to figure out. See the fixup.

**Review by cassava — APPROVED** [review]

**Comment by cassava** [comment]
Rebase and then LGTM.

**Pull request merged**